lib/sepa_king/transaction.rb

```diff
@@ -4,6 +4,8 @@ class Transaction
     include ActiveModel::Validations
     extend Converter
 
+    DEFAULT_REQUESTED_DATE = Date.new(1999, 1, 1).freeze
```

Comment:
Thinking about it: it is not the Next possible date (that would be Date.tomorrow / Date.today.next) and earliest possible date is not correct either, because the day after 1999-01-01 will not work either.

It may be stupid but after letting this sink in I don't fully agree with both suggestions for the variable name ;-) In fact, it is the default requested date and not the erliest possible date (which is different for direct debit and transfer), so maybe the constant should be named after it?